### PR TITLE
CAT-1912 : Adding new getters and setters for stripes & not allowing change of stripes for an existing lvm

### PR DESCRIPTION
**Pull request by malikparvez**
## Summary
Adding new getters and setters for stripes.

## Additional Context
Add any additional context about the problem here. 
- [ ] Root cause and the steps to reproduce. (If applicable)
- [ ] Thought process behind the implementation.

## Related Issues (if any)
Mention any related issues or pull requests.

## Checklist
- [ ] 🟢 Spec tests.
- [ ] 🟢 Acceptance tests.
- [ ] Manually verified. (For example `puppet apply`)

### CODE REVIEW

**Review by Ramesh7 — APPROVED** [review]

File: lib/puppet/provider/logical_volume/lvm.rb [hunk 1]
```diff
@@ -221,6 +221,22 @@ def size=(new_size)
...
 
+  def stripes
+    # Run the lvs command with the -o option to get only the stripes count
+    raw = (lvs '-o', 'stripes', '--noheadings', path)
+
+    output_array = raw.strip
+    output_array
+  end
+
+  def stripes=(new_stripes_count)
```
Comment (bastelfreak) [inline lib/puppet/provider/logical_volume/lvm.rb:232]: what's the purpose of this setter? Is that used anywhere?

Comment (malikparvez) [reply lib/puppet/provider/logical_volume/lvm.rb:232]: Hey @bastelfreak , I was trying to not allow the user to change the stripes of an existing lvm so added setter to raise an error. Referred the implementation the way mirror & mirror= getter & setter handled. Please let me know if there is better way to do this happy to comply...

Comment (bastelfreak) [reply lib/puppet/provider/logical_volume/lvm.rb:232]: ah okay. that was my impression as well. But than why did you change it to a gettter and setter at all?

Comment (malikparvez) [reply lib/puppet/provider/logical_volume/lvm.rb:232]: I could only think of this way to handle it, can you suggest how to do this without getter and setter?

**Review by bastelfreak — APPROVED** [review]

File: lib/puppet/provider/logical_volume/lvm.rb (hunk 1 above)
Comment (malikparvez) [reply lib/puppet/provider/logical_volume/lvm.rb:232]: Discussed in DM, hence resolving it...

**Pull request merged**